client/landing/stepper/declarative-flow/internals/steps-repository/free-setup/styles.scss

```diff
@@ -0,0 +1,16 @@
+@import "../style";
+
+$font-family: "SF Pro Text", $sans;
+
+.free-setup {
+	.step-container__content {
+		display: flex;
+		justify-content: center;
+		margin-bottom: 30px;
+		min-height: unset;
+	}
+
+	.free-setup .step-container__header {
```

Comment:
Is this intended? it looks like we are looking for `.free-setup` inside `.free-setup` ?